storage/src/journal/mod.rs

```diff
@@ -15,7 +15,7 @@
 //!
 //! ```text
 //! +---+---+---+---+---+---+---+---+---+---+---+
-//! | 0 | 1 | 2 | 3 |    ...    | 8 | 9 |10 |11 |
+//! | 0 | 1 | 2 | 3 |    ...    | N |N+1|N+2|N+3|
```

Comment:
I think this is a fair suggestion. Would you mind indicating below `C = CRC32(Data)` what `N` is?

Wonder if it would be clear to just call it `D` (for Data) and do `D+4, D+5, D+6, D+7` for size (incorporating values from before)